src/esportsbot/cogs/DefaultRoleCog.py

```diff
@@ -11,88 +11,81 @@ def __init__(self, bot):
+                    # Obtain role object from the guild to check it exists
+                    role_obj = ctx.author.guild.get_role(cleaned_role_id)
+                    # Add role to array to add post checks
+                    checked_roles.append(cleaned_role_id)
+                except Exception as err:
+                    print(err)
+                    checking_error = True
+            if not checking_error:
+                for role in checked_roles:
+                    DBGatewayActions().create(Default_roles(guild_id=ctx.author.guild.id, role_id=role))
+                await ctx.channel.send(self.STRINGS['default_roles_set'].format(roles=args))
+            else:
+                await ctx.channel.send(self.STRINGS['default_roles_set_error'])
 
     @commands.command(
-        name="getdefaultrole",
+        name="getdefaultroles",
         usage="",
-        help="Gets the role that the server gives to members when they join the server"
+        help="Gets the roles that the server gives to members when they join the server"
     )
     @commands.has_permissions(administrator=True)
-    async def getdefaultrole(self, ctx):
-        guild = DBGatewayActions().get(Guild_info, guild_id=ctx.author.guild.id)
-        if not guild:
-            await ctx.channel.send(self.STRINGS['default_role_missing'])
-            return
-
-        if guild.default_role_id:
-            await ctx.channel.send(self.STRINGS['default_role_get'].format(role_id=guild.default_role_id))
+    async def getdefaultroles(self, ctx):
+        # Get all the default role for the server from database
+        guild_default_roles = DBGatewayActions().list(Default_roles, guild_id=ctx.author.guild.id)
+        # Check to see if any roles exist
+        if guild_default_roles:
+            # Create list of roles from database response
+            apply_roles = [ctx.author.guild.get_role(role.role_id) for role in guild_default_roles]
+            # Return all the default roles to the user
+            await ctx.channel.send(self.STRINGS['default_role_get'].format(role_ids=(' '.join(f'<@&{x.id}>' for x in apply_roles))))
         else:
             await ctx.channel.send(self.STRINGS['default_role_missing'])
 
     @commands.command(
-        name="removedefaultrole",
+        name="removedefaultroles",
         usage="",
-        help="Removes the role that the server gives to members when they join the server"
+        help="Removes the roles that the server gives to members when they join the server"
     )
     @commands.has_permissions(administrator=True)
-    async def removedefaultrole(self, ctx):
-        guild = DBGatewayActions().get(Guild_info, guild_id=ctx.author.guild.id)
-        if not guild:
-            await ctx.channel.send(self.STRINGS['default_role_missing'])
-            return
-
-        if guild.default_role_id:
-            guild.default_role_id = None
-            DBGatewayActions().update(guild)
+    async def removedefaultroles(self, ctx):
```

Comment:
Similar to the setdefaultrole command, could make use of `discord.Context.message.role_mentions` so that you can remove individual roles from the list instead of totally clearing it.